### DISCUSSION TITLE
How to connect/send logs from cloud vms to OnPrem Security Onion

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#5327] [Unsupported Versions] [answered]
Hello, I was wondering how I would be able to send logs from an Azure vm to my onprem security onion. I understand you have to install filebeat on the vm and then point it towards elastic/logstash, but is there a specific way to get the cloud vm to communicate with the network that my OnPrem security onion is on?

**Comment by weslambert** [comment, marked as answer]
You would need to setup VPN-like connectivity for internal <-> internal communication.
